modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/IgniteTransactionsImpl.java

```diff
@@ -35,14 +35,14 @@
 public class IgniteTransactionsImpl implements IgniteTransactions {
     private final TxManager txManager;
 
-    private final HybridTimestampTracker observableTimestampTracker;
+    private final HybridTimestampTrackerImpl observableTimestampTracker;
```

Comment:
let's use interface instead of implementation. Besides, there are number of methods which are not used anymore. Let's clean up `IgniteTransactionsImpl`

Reply:
In this instance, the field can only be an instance of HybridTimestampTrackerImpl.
It is not worth overdoing with the use of interfaces.

Reply:
> there are number of methods which are not used anymore
Good point.